docs: run a specific test (#257)
GreptimeTeam/docs: docs/developer-guide/tests/sqlness-test.md

```
It automatically finishes the following procedures: compile `GreptimeDB`, start it, grab tests and feed it to
the server, then collect and compare the results. You only need to check whether there are new `.output` files.
If not, congratulations, the test is passed 🥳!

### Run a specific test

```shell
cargo sqlness your_test
```

> If you specify a second argument, only test cases containing the specified string in their names will be executed.

Reviewer: ```suggestion
If you specify a second argument, only test cases containing the specified string in their names will be executed. Sqlness also supports filtering based on environment. The filter is accepted as a regex string and the case name will be examined in the format of `env:case`.
```

> If the second argument was specified, only run cases containing the second argument string in their names.

Reviewer: ```suggestion
If you specify a second argument, only test cases containing the specified string in their names will be executed.
```

> If you specify a second argument, only test cases containing the specified string in their names will be executed.

Reviewer 2: and sqlness also support filter on env. Filter is accepted as regex string. And case name will be examine in form of `env:case`.

Reviewer: @waynexia, do we need to include this statement in the doc?

Reviewer 2: Yes, please. I rephrase it to:

sqlness also supports filtering based on environment. The filter is accepted as a regex string and the case name will be examined in the format of 'env:case'.
